src/postgres/commands/createPostgresServer/steps/PostgresServerSkuStep.ts

```diff
@@ -40,9 +40,11 @@ export class PostgresServerSkuStep extends AzureWizardPromptStep<IPostgresServer
         return wizardContext.sku === undefined;
     }
 
-    public async getPicks(): Promise<IAzureQuickPickItem<Sku | undefined>[]> {
-        const options: IAzureQuickPickItem<Sku | undefined>[] = [];
-        availableSkus.forEach((option) => {
+    public async getPicks(serverType: PostgresServerType): Promise<IAzureQuickPickItem<AbstractSku | undefined>[]> {
+        const options: IAzureQuickPickItem<AbstractSku | undefined>[] = [];
+        const skuOptions: ISkuOption[] = serverType == PostgresServerType.Single ? singleServerSkus : flexibleServerSkus ;
```

Comment:
Always use `===` instead of `==`

I'm kind of surprised our linting doesn't catch this - thoughts @wwlorey
https://eslint.org/docs/rules/eqeqeq